core/src/com/unciv/models/UncivSound.kt

```diff
@@ -12,5 +15,14 @@ enum class UncivSound(val value: String) {
     Policy("policy"),
     Paper("paper"),
     Whoosh("whoosh"),
-    Silent("")
+    Silent(""),
+    Custom("");
+    companion object {
+        fun getCustom(name: String?): UncivSound {
+            if (name == null || name.isEmpty()) return Click
+            val item = Custom
```

Comment:
I'm like 80% sure that this means every getCustom will return THE SAME INSTANCE of Custom, but just set one of its parameters differently. This means that if you have multiple custom sounds, the last initialized one overrides the rest. This seems an unsafe way of doing things,

Reply:
Make that 100% - I said so it's essentially a single global var...